packages/psss-server/src/connections/MeditrakConnection.ts

```diff
@@ -3,10 +3,112 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
+import { generateId } from '@tupaia/database';
```

Comment:
I believe this is a safe way of generating id's for creating new survey responses / answers?

Reply:
Yeah it is...a bit rough making psss-server generate ids though! Can you make a psss-server tech debt card and add this to it, for addressing later?

Reply:
https://github.com/beyondessential/tupaia-backlog/issues/1755